animations/index.md

```diff
@@ -1,159 +1,95 @@
+ <br>
+ For more information including a list of implicit widgets, see [Implicit animations](/animations/implicit_widgets).
+
+
+### Transition animation widgets
+
+*Transition animations*   NEED - description of transition widgets - what are they and why would you use transition instead of implicit or explicit animations.
+
+<br>
+For more information including a list of transition widgets, [Transition animations](/animations/transition_widgets).  
+
+
+### Explicit animation widgets
+
+ *Explicit animations* involve building customized animations using the `AnimationBuilder` widget that allows you to customize many of the animation elements that are pre-defined in the implicit and transition animation widgets. When you build explicit animations, you manually add listeners, tickers, and other elements for customization.  
+ <br>
+ For more information, see [Explicit animations](/animations/explicit_widgets).  
+
+## Choosing an animation widget
+Choosing an animation widget depends on the customization that you need for your animation and whether there is an existing widget available for reuse. The table below shows the customizable elements in each animation type.
+
+![Choosing an animation table](images/animation_table4.png)
+<br>
+
+
+## Flutter packages
+The Flutter SDK includes several libraries that contain pre-defined animation widgets. To access the Flutter animation widgets, determine which library contains the widget that you want to use and then add the appropriate import statement to the `main.dart` file.
+
+```Dart
+import 'package:flutter/animation.dart';
+import 'package:flutter/material.dart';
+```
+For information on the SDK libraries and available widgets in each library, see the following resources.
+
+* [Flutter SDK libraries](https://docs.flutter.io/index.html)  
+* [Animation and Motion Widgets](/widgets/animation/)  
+* [Flutter Widget Index](/widgets/widgetindex/)  
+* [Widget Catalog](/widgets/)
+
+## Flutter animation resources  
+To learn about Flutter animations, check out the following resources.
+
+### Concepts  
+[Flutter Animation Concepts](/animations/concepts) includes information about how Flutter animation works and definitions of the main animation concepts. You'll find information on the main animation widgets such as the `AnimationController` or `AnimationBuilder`, and information about tweens, curves, tickers, and more.  
+
+
+### Animation and Motion widgets  
+For a list of Flutter animation widgets, see the following:
+* [Implicit animation widgets](/animations/implicit_widgets)  
+* [Transition animation widgets](/animations/transition_widgets)  
+* [Explicit animation widgets](/animations/explicit_widgets)  
+* [Animation and Motion Widgets Catalog](/widgets/animation/)  
+
+
+### Common animation patterns  
```

Comment:
We have great content here. I would move this section up, perhaps before Flutter packages. I would also show animated illustrations next to each pattern listed below. We already have the assets in each subpage.

Reply:
Great comment - although at this point there are already several animations on the page. I think the page is getting way to busy - and should probably be separated into some subpages - one page would be about the widget types - another would be about the animation patterns - another page would list out the additional resources such as the tech video pages, articles, tutorials, codelabs, etc.

my 2cents: segmenting information helps to provide information in small clips - pardon the animation pun - but adding all the information on the first page presents information that is not organized in a logical manner